plugins/plugin-pullrequest-parent/che-plugin-pullrequest-ide/src/main/java/org/eclipse/che/plugin/pullrequest/client/steps/InitializeWorkflowContextStep.java

```diff
@@ -100,7 +100,24 @@ public void apply(final List<Remote> remotes) throws OperationException {
         };
     }
 
-    private String getContributeToBranchName(final ProjectConfig project) {
+    protected void setContributeToBranchName(Context context) {
+        String contributeToBranchName = getBranchFromProjectMetadata(context.getProject());
+
+        if (contributeToBranchName != null) {
+            context.setContributeToBranchName(contributeToBranchName);
+            return;
+        }
+
+        vcsServiceProvider.getVcsService(context.getProject()) //
```

Comment:
>Can you elaborate what's wrong with that ?

Nothing. It's just looks weird to add your personal preference formatting and to be
surprised that other committees may not like it.

About formatting.
This is a formatting style for intellij. Developed by @florent and @evoevodin
https://github.com/codenvy-legacy/dev/blob/cded88bbe9b6cf006ac5ba5b99ca9069a87109cc/che-codestyle/src/main/resources/che-codestyle-intellij-new.xml

Docker command to run without Intellij
```
docker run -ti \
            -u root \
            --net="host" \
            --privileged=true \
            -e DISPLAY=${DISPLAY} \
            -v /tmp/.X11-unix:/tmp/.X11-unix \
            -v ${HOME}/.Xauthority:/home/developer/.Xauthority \
            -v  {path to sources}:/cheformat/sources ksmster/che-format
```
If you don't like this format we can discuss it in che-dev mailist.